src/modules/rlm_files/rlm_files.c

```diff
@@ -214,6 +214,18 @@ static int getusersfile(TALLOC_CTX *ctx, char const *filename, rbtree_t **ptree,
 			 *	worth doing.
 			 */
 			for (vp = fr_cursor_init(&cursor, &entry->reply); vp; vp = fr_cursor_next(&cursor)) {
+				/*
+				 * If we're dealing with Tunnel-Password,
+				 * note that we're truncating it to 249 bytes.
+				 */
+				if (strcmp(vp->da->name, "Tunnel-Password") == 0) {
+					if (strlen(vp->data.strvalue) > 249) {
```

Comment:
As an aside, I think this is technically wrong as we'd likely need to handle the format like:

```
    Tunnel-Password = <something-short>
    Tunnel-Password += <something-long-but-under-249>
```

But that seems like an edge case. If I had a better place to check the computed value of `Tunnel-Password` on startup, that'd be nicer and more correct.

Reply:
The above snippet is for two tunnel passwords, not for two strings which add up to >249 characters.